libs/clients/charge-fjs-v2/src/lib/chargeFjsV2Client.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from '@island.is/nest/config'
+import * as z from 'zod'
+
+const schema = z.object({
+  xRoadServicePath: z.string(),
+  fetchTimeout: z.number().int(),
+  tokenExchangeScope: z.array(z.string()),
+  requestActorToken: z.boolean(),
+})
+
+export const ChargeFjsV2ClientConfig = defineConfig({
+  name: 'ChargeFjsV2Client',
+  schema,
+  load(env) {
+    return {
+      xRoadServicePath: env.required(
+        'XROAD_CHARGE_FJS_V2_PATH',
+        'IS-DEV/GOV/10021/FJS-Public/chargeFJS_v2',
+      ),
+      fetchTimeout: env.optionalJSON('XROAD_CHARGE_FJS_V2_TIMEOUT') ?? 20000,
+      tokenExchangeScope: env.optionalJSON('XROAD_CHARGE_FJS_V2_SCOPE') ?? [
+        '@fjs.is/finance',
+        // TODO: Remove when fjs has migrated to the scope above.
+        'api_resource.scope',
```

Comment:
Is this a new API from FJS that is still using the `api_resource.scope`?

Reply:
I talked with Ágúst Leó and he told me to use @fjs.is/finance, but is soon going to go over these scopes.
So I just assumed I should set it up the same as for Finance service for now?

Reply:
Okay sounds like they have at least started to use the new scope for new APIs. So we should remove the `api_resource.scope`